internal/pkg/template/templates/workloads/partials/cf/autoscaling.yml

```
@@ -113,8 +113,21 @@ AutoScalingPolicyECSServiceAverageCPUUtilization:
     TargetTrackingScalingPolicyConfiguration:
       PredefinedMetricSpecification:
         PredefinedMetricType: ECSServiceAverageCPUUtilization
+      {{if .Autoscaling.CPUCooldown}}
```

```suggestion
      {{- if .Autoscaling.CPUCooldown}}
```
same for the else and end below

---

(it's to trim whitespace https://pkg.go.dev/text/template#hdr-Text_and_spaces)